routes/drones.js

```diff
@@ -51,8 +51,9 @@ router.post('/drones/:id/edit', (req, res, next) => {
 })
 
 router.post('/drones/:id/delete', (req, res, next) => {
-  // Iteration #5: Delete the drone
-  // ... your code here
+  Drone.findByIdAndDelete(req.params.id).then(() => {
+    res.redirect('/drones')
+  })
 })
```

Comment:
Always have a `catch` , it specifies exact location where error occured - if it happens